main.go

```diff
@@ -118,6 +117,45 @@ func createRepo(args []string) {
 	getRepo([]string{repoPath})
 }
 
+func deleteRepo(args []string) {
+	var repostr, confirmation string
+
+	if len(args) == 0 {
+		util.Die(usage)
+	} else {
+		repostr = args[0]
+	}
+
+	repocl := repo.NewClient(util.Config.RepoHost)
+	err := repocl.LoadToken()
+	util.CheckError(err)
+
+	repoinfo, err := repocl.GetRepo(repostr)
+	util.CheckError(err)
+
+	if repoinfo.FullName != repostr {
+		util.LogWrite("ERROR: Mismatch in repository names: %s != %s", repoinfo.FullName, repostr)
+		util.Die("An unexpected error occurred while communicating with the server.")
+	}
+
+	fmt.Println("--- WARNING ---")
+	fmt.Println("You are about to delete a remote repository, all its files, and history.")
+	fmt.Println("This action is irreversible.")
+
+	fmt.Println("If you are sure you want to delete this repository, type its full name (owner/name) below")
```

Comment:
The web ui tells you exactly what to type. this is a bit less straight forward (maybe good)